src/frontend/js/components/ReferralDetailAssignment/AssignUnitModal.spec.tsx

```diff
@@ -0,0 +1,161 @@
+import { act, render, screen, waitFor } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { Unit } from 'components/Unit';
```

Comment:
should be in the second group of imports along with other imports from our own code